### DISCUSSION TITLE
Individual PP skill ratings on profiles

### DISCUSSION THREAD

**Discussion by Pikachuboy2275** [ppy/osu#17105] [Ideas] [answered]
How about something similar to osuskills.com for osu! profiles?

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/9463

**Reply by Pikachuboy2275** [reply]
I meant profile stats

**Reply by smoogipoo** [reply]
Right, well that's still something that we'll eventually want to happen:
![image](https://user-images.githubusercontent.com/1329837/157388253-dfe7b8ca-fc28-4c53-8594-6193dc217d00.png)
